extensions/wikia/AdEngine/js/spec/AdContext.spec.js

```diff
@@ -672,5 +673,15 @@ describe('AdContext', function () {
 
 		getModule();
 		expect(mocks.geo.isProperGeo.calls.count()).toEqual(1);
-	})
+	});
+
+	it('checks which lABrador keyvals should be sent to DFP', function () {
+		mocks.instantGlobals = {
+			wgAdDriverRubiconDfpCountries: ['XX/50'],
+			wgAdDriverLABradorDfpKeyvals: ['wgAdDriverRubiconDfpCountries_A_50:rub-dfp-test']
+		};
+
+		getModule();
+		expect(getModule().get('opts.labradorDfp')).toEqual('rub-dfp-test');
+	});
```

Comment:
What do you think about adding test covering cases:

- empty array is passed as argument;
- null/undefined are passed as arguments?